base/cvd/cuttlefish/host/commands/cvd/fetch/BUILD.bazel

```diff
@@ -38,6 +38,9 @@ cc_library(
         "@jsoncpp",
         "@rules_cc//cc/runfiles",
     ],
+    data = [
+        "//cuttlefish/host/commands/logcat_receiver",
```

Comment:
Just curious, if the `logcat_receiver` binary is installed in `/usr/lib/cuttlefish-common/bin/logcat_receiver` as part of the debian package installation why do we need to have it as data in the `fetch` cc_library as well?

Reply:
In the local development case, this makes it possible to edit files in the repository and `bazel run cvd -- fetch` to include the latest changes. If we use files in `/usr/lib/cuttlefish-common/bin/`, the only way to locally validate changes to our host code is to `debuild` the entire package and install it with `dpkg -i`.